README.md

```diff
@@ -273,3 +274,19 @@ sudo usermod -aG docker $USER
 ```
 
 After adding your user to the `docker` group, it is necessary to start a new bash session for the changes to take effect.
+
+### Temporary Directory Creation Error
+
+This error occurs because the `das-cli` executable includes a compressed support file that is unpacked into a temporary directory during execution. When the storage is full, the extraction process fails.
+
+#### Solution
```

Comment:
Explaining  how to free disk space is totally unnecessary in the context of our users. Instead of this explanation, you should just provide the amount of disk space that's expected to be required, which is a much more useful information.